amperity_datagrid/source/destination_meta_ads_manager.rst

```diff
@@ -1369,6 +1369,16 @@ The fields are listed alphabetically, but may be returned by a query in any orde
        **website**
          Use when the offline conversion was made on a website.
 
+         When **action_source** is set to **website** the following fields are required: **client_user_agent**, **event_id**, and **event_source_url**. These fields must be in the results that are sent to |destination-name|; missing or empty values are filtered from the results.
+
+         * The value for **client_user_agent** must be the user agent for the browser corresponding to the event.
+
+         * The value for **event_id** is a unique advertiser ID.
```

Comment:
i would tweak this a bit. this is what facebook's documentation says:

> This ID can be any unique string chosen by the advertiser.

is not an advertiser id per se but just any value they choose.